codecov.yml

```diff
@@ -0,0 +1,53 @@
+codecov:
+  branch: master
+  bot: "mikegray"
+  ci:
+    - "jenkins.noms.io"
+
+coverage:
+  precision: 2         # how many decimal places to display in the UI: 0 <= value <= 4
+  round: down          # how coverage is rounded: down/up/nearest
+  range: 60...100      # custom range of coverage colors from red -> yellow -> green
+
+  notify:
+    slack:
+      default:
+        url: "secret:n+BYhIXTXsaCiMKB3vOf6yP68ytdKd3WpXtJFWPEUsEWXDiGnU5dTB5DO2yv8tR0COdxvs7K31hVpEfHEXdoXOaQhUw3FKf3fh8KZDLN7CGTbeDhw1uNGGyBr2d2TWnopzYtcXomdwMmuckARtiWQx0YXJiZY9YyCrIoDK9HIJQ="
```

Comment:
Yes, I generated it via the codecov.io website so only they know how to decrypt it to determine the real URL to use.